narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2453 @@
+
+<!-- This is the right-side text -->
+
+```auspiceMainDisplayMarkdown
+<img src="https://github.com/nextstrain/ncov/raw/master/figures/local-spark-expansion.jpeg" width="70%">
+
+"COVID-19 in Wuhan went from an index case in ~Nov 2019 to several thousand cases by mid-Jan 2020, thus going from initial seeding event to widespread local transmission in the span of ~10 weeks. We believe that international seeding events started to occur in mid-Jan. Thus we have a critical ~10 weeks from then to late-March to contain these nascent outbreaks before they become sizable." [- Trevor Bedford](https://twitter.com/trvrb/status/1226241284207038464), Feb 2020
+```
+
+<!-- ############ SLIDE BREAK ############# -->
+# [Once introduced, epidemics become evident ~4-8 weeks later](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington&label=clade:B1&p=grid&r=location)
+
+We can see a clear example of this in Washington state.
+Mousing over the internal nodes at the root of the clade tells us that the virus was likely introduced once to this area in early- to mid-January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)).
+<br><br>
+Now, in mid-March (~8 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  
+<br>
+<img src="https://github.com/nextstrain/ncov/raw/master/figures/wa_doubling-rate_2020-03-19.png" width="70%">
+
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 9-->
+# [How outbreaks grow: an example](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington,Utah&label=clade:B1&p=grid)
+
+We can also see early evidence that 'sparks' from this cluster have led to introductions in Utah. Following this pattern, we expect to see a local outbreak in Utah pick up within the next 4 weeks.
+<br><br>
+This is just one example. Most importantly, areas that aren't yet aware of local outbreaks should prepare now and start surveillance testing.  
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 10 -->
+# [Mitigation measures take time, but they save lives](https://nextstrain.org/ncov/2020-03-19)
+
+Following this same logic, there are likely many local transmission chains that we don't yet know about.
+<br><br>
+This means that even for areas that just implemented social distancing measures, we should still expect case counts to grow over the next few weeks.
+These increases do NOT indicate that these measures aren't working. It just takes time for already-infected individuals (and possibly their household members) to manifest symptoms, be treated, and recover. We'll also see a bump in the number of reported cases as testing becomes more widely available.
```

Comment:
Replace: "We'll also see a bump in the number of reported cases as testing becomes more widely available."
With: "We also expect to see a bump in the number of reported cases as testing becomes more widely available."